docs/en/latest/admin-api.md

```diff
@@ -657,12 +659,32 @@ After the execution is successful, nodes will not retain the original data, and
 
 ```
 
+Example 2: Create upstream and configure the scheme as `https`.
```

Comment:
This is an example of proxying to upstream https

Reply:
updated.